app/ui/src/test/java/com/fsck/k9/fragment/AccountRetrieverTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.fsck.k9.fragment
+
+import android.database.Cursor
+import com.fsck.k9.Account
+import com.fsck.k9.Preferences
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import org.mockito.ArgumentMatchers.anyInt
+import org.mockito.ArgumentMatchers.anyString
+import org.mockito.BDDMockito.given
+import org.mockito.Mockito.mock
+
+class AccountRetrieverTest {
+
+    @Test fun shouldRetrieveAccountCorrectly() {
+        // given
```

Comment:
We use empty lines between the 'given', 'when', and 'then' blocks. That way we don't need comments.

Reply:
Outdated for now, but will remember for the future.